src/components/LandingPage/LandingPage.css

```
@@ -0,0 +1,101 @@
+#landing-page-container {
+    display: flex;
+    justify-content: center;
+    max-width: 100%;
+}
+
+#info-section {
+    display: flex;
+    flex-direction: column;
+    max-width: 40%;
+    flex: 1;
+    margin-top: 5%;
+    margin-right: 3%;
+}
+
+#title {
+    font-weight: 700px;
+    color: #000000;
+    font-size: 54px;
+    font-style: normal;
+    letter-spacing: 0.035em;
+    line-height: 56px;
+    display: flex;
+    text-align: left;
+}
+
+#lab-description {
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 24px;
+    letter-spacing: 0.04em;
+    display: flex;
+    text-align: left;
+    margin-top: 15px;
+}
+
+.btn_ubc_home {
+    display: flex;
+    width: 219px;
+    height: 42px;
+    background-color: #77B0F3;
+    font-weight: 700;
+    letter-spacing: 0.04em;
+    font-size: 15px;
+    line-height: 19px;
+    color: #ffffff;
+    text-align: center;
+    padding-bottom: 6px;
+    justify-content: center;
+    margin-top: 20px;
+}
```

For this part here, instead of adding the styling to the <a> tag directly in the `LandingPage.tsx` file (line 18), you can try the code below. This can help provide a clear distinction between the link background vs its text, and can also help with `hover` functionality: 😃  

```suggestion
.btn_ubc_home {
    display: flex;
    width: 219px;
    height: 42px;
    background-color: #77B0F3;
    justify-content: center;
    margin-top: 25px;
}

a {
    font-weight: 700;
    letter-spacing: 0.04em;
    font-size: 15px;
    line-height: 19px;
    color: #ffffff;
    text-align: center;
    text-decoration: none;
    padding: 10px;
}

a:hover {
    text-decoration: underline;
}
```